<launch>

  <!--- Run Rplidar -->
  <node name="rplidarNode"          pkg="rplidar_ros"  type="rplidarNode" output="screen">
    <param name="serial_port"         type="string" value="/dev/rplidar"/>  
    <param name="serial_baudrate"     type="int"    value="460800"/>
    <param name="frame_id"            type="string" value="laser"/>
    <param name="inverted"            type="bool"   value="false"/>
    <param name="angle_compensate"    type="bool"   value="true"/>
    <param name="scan_frequency"      type="double" value="10.0"/>
    <param name="scan_mode"           type="string" value="Standard"/>
    <remap from="scan" to="scan_raw"/>
  </node>

  <!-- Run lidar filter -->
  <node pkg="wpr2_bringup" type="wpr2_lidar_filter" name="wpr2_lidar_filter" output="screen">
    <param name="pub_topic" value="/scan"/>
  </node>

</launch>
